packages/helpers/src/is-chat-channel.ts

```diff
@@ -1,14 +1,19 @@
-import { Participant, ChannelData } from '@serge/custom-types'
+import { PARTICIPANT_CUSTOM } from '@serge/config'
+import { ChannelData } from '@serge/custom-types'
+import { CoreParticipant, ParticipantCustom } from '@serge/custom-types/participant'
 
 const isChatChannel = (channel: ChannelData): boolean => {

```

Comment:
Soon, this will get tidier, as we introduce "Add > Chat Channel" to the admin page, which will create an instance of `ChannelChat`.  A custom editor page will look like `Channel-Custom`, but it won't have the templates drop-down.